_tools/lint_plugin.ts

```diff
@@ -38,5 +38,32 @@ export default {
         };
       },
     },
+    // https://docs.deno.com/runtime/contributing/style_guide/#do-not-depend-on-external-code.
+    "no-external-code": {
+      create(context) {
+        return {
+          ImportDeclaration(node) {
+            const resolvedSpecifier = import.meta.resolve(node.source.value);
+            if (
+              resolvedSpecifier.startsWith("file:") ||
+              resolvedSpecifier.startsWith("jsr:@std") ||
+              resolvedSpecifier.startsWith("jsr:/@std") ||
+              resolvedSpecifier.startsWith("jsr:@deno") ||
+              resolvedSpecifier.startsWith("jsr:/@deno") ||
+              resolvedSpecifier.startsWith("node:")
```

Comment:
I've extended this rule so that it makes more sense when used within other Deno codebases to also include `@deno`-scoped and Node imports. If this seems reasonable, I think we should also rewrite [the rule](https://docs.deno.com/runtime/contributing/style_guide/#do-not-depend-on-external-code.) to match. What do we think?

Reply:
It makes sense to include `node:` import as it's builtin to Deno and also it's already used a lot in `@std/fs`. I don't think std should rely on `@deno` namespace, especially in published packages. I think most `@deno`-scoped packages are used in `_tools`. How about excluding `_tools` scripts from lint target?